src/front/spv/mod.rs

```diff
@@ -111,6 +111,7 @@ pub enum ModuleState {
     MemoryModel,
     EntryPoint,
     ExecutionMode,
+    String,
```

Comment:
this shouldn't be a separate module state, it's the same state as `Source` in the spec